website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,55 @@
+## Usage
+
+### When to use
+
+- To reveal content as needed while keeping the UI organized and visually manageable.
+- To hide long pieces of content with a similar structure.
+
+### When not to use
+
+- To hide basic non-critical information to users. Instead, consider using a [Reveal](/components/reveal).
```

Comment:
Might be helpful to include a point in the "When to use" covering the difference in importance of content between the Reveal and the Accordion. That clear up the distinction between these two components.

Reply:
I liked this idea. I added this information as a subsection. Let me know if the format and the content read well.